po/common/common.pot

```diff
@@ -7471,6 +7471,22 @@ msgctxt "scan"
 msgid "Scan"
 msgstr "Scan"
 
+msgctxt "reset_preferences_details_beauty"
+msgid "Organic farming and fair trade"
+msgstr "Organic farming and fair trade"
+
+msgctxt "reset_preferences_details_food"
+msgid "Nutri-Score, Green-Score and food processing level (NOVA)"
+msgstr "Nutri-Score, Green-Score and food processing level (NOVA)"
+
+msgctxt "reset_preferences_details_product"
+msgid "Nutri-Score, Green-Score and food processing level (NOVA)"
+msgstr "Nutri-Score, Green-Score and food processing level (NOVA)"
+
+msgctxt "reset_preferences_details_pet_food"
```

Comment:
for pet food, we just have organic and fair trade:

![image](https://github.com/user-attachments/assets/7c22d6c8-fa42-452a-bc19-d500b2e327d0)